server/tests/trim_gap_handling.rs

```diff
@@ -133,11 +133,18 @@ async fn fast_forward_over_trim_gap() -> googletest::Result<()> {
         .create_partition_snapshot(CreatePartitionSnapshotRequest { partition_id: 0 })
         .await?
         .into_inner();
+    info!(
+        "Snapshot created up to at least LSN {}",
```

Comment:
"at least" might be misleading. Perhaps "that include LSN or higher"? Alternatively, describe it terms of what it doesn't have, i.e. "include changes up to LSN"